src/Microsoft.AspNetCore.Hosting/Internal/ActivityTrackingOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Hosting.Internal
+{
+    public class ActivityTrackingOptions
+    {
```

Comment:
Needs its own off switch. Right now the only on/off control you get is linked to an unrelated diagnostic source.